cmd/snap-bootstrap/partition/sfdisk.go

```diff
@@ -93,8 +93,9 @@ type DeviceLayout struct {
 	// size in bytes
 	Size gadget.Size
 	// sector size in bytes
-	SectorSize     gadget.Size
-	partitionTable *sfdiskPartitionTable
+	SectorSize        gadget.Size
+	CreatedPartitions []string
```

Comment:
Could we add a `Created bool` to `DeviceStructure` ?

Reply:
This was initially used and then discarded in a previous PR because we were not sure it's really needed, but this seems to be a good application for that.